### ISSUE TITLE
Readiness probe fails with redis:6.2.6 and no auth

### ISSUE THREAD

**Issue by grubenhund** [spotahome/redis-operator#331]
### Expected behaviour

Create redis failover without authentication on redis 6.2.6

### Actual behaviour

Rfr pods are never ready because /redis-readiness/ready.sh fails to get replication info when env REDIS_PASSWORD is empty.

### Steps to reproduce the behaviour

Create redis failover cr with image redis:6.2.6-alpine or change redis image from redis:6.2.5-alpine (or prior) to redis:6.2.6-alpine 

### Environment

How are the pieces configured?
* Redis Operator image 3.2-alpine
* Kubernetes v1.21
* no RBAC

### Logs
6.2.5:

```
root@999d2783164c:/data# redis-cli -a "${REDIS_PASSWORD}" info replication 
Warning: Using a password with '-a' or '-u' option on the command line interface may not be safe.
Warning: AUTH failed
\# Replication
role:master
connected_slaves:0
master_failover_state:no-failover
master_replid:6b92a65eb2961b77f9574d9ef632edae7a8026c2
master_replid2:0000000000000000000000000000000000000000
master_repl_offset:0
second_repl_offset:-1
repl_backlog_active:0
repl_backlog_size:1048576
repl_backlog_first_byte_offset:0
repl_backlog_histlen:0
```

6.2.6:
```
root@f1c3f8b45004:/data# redis-cli -a "${REDIS_PASSWORD}" info replication
Warning: Using a password with '-a' or '-u' option on the command line interface may not be safe.
AUTH failed: ERR AUTH <password> called without any password configured for the default user. Are you sure your configuration is correct?
```

**Comment by ArneSchoonvliet** [comment]
For reference sake I already provided a potential fix as pull request. 
#330

**Comment by ese** [comment]
Fixed in #330
